src/url-replacements.js

```diff
@@ -239,6 +239,14 @@ class UrlReplacements {
       return this.getTimingData_('navigationStart',
           'domContentLoadedEventStart');
     });
+
+    // Returns an identifier for the viewer.
+    this.set_('VIEWER', () => {
+      return viewerFor(this.win_).getViewerOrigin().then(origin => {
+        const url = parseUrl(origin);
+        return (url && url.hostname) || '';
```

Comment:
One thing to consider - we can return origin for `VIEWER` var, e.g. not `www.google.com` but `https://www.google.com`. LMK your thoughts.